### ISSUE TITLE
allow list of images (not paths) to be specified as command arguments

### ISSUE THREAD

**Issue by calestyo** [BestImageViewer/geeqie#1323]
**Setup (please complete the following information):**
 - Geeqie version [geeqie --version]: Geeqie 2.4 GTK3

**Describe your feature request**
It would be nice if there was an option, that makes `geeqie` read a (possibly long) list of arguments that are path to image files (not directories containing such files) and display them as if they were in one directory (i.e. so that one can scroll through them, sees them all in one thumbnail list, etc.).

The benefit of having this would be that one could use it e.g. with `find` and `xargs`, something like:
```
find . -ctime -2 -type f \( -name '*.png' -o -name '*.jpg' -o -name '*.jpeg' -o -name '*.gif' \) -print0  |  xargs --null --no-run-if-empty -- geeqie
```

Thanks,
Chris.

**Comment by caclark** [comment]
Your command (or maybe with `geeqie --list`) will open the files as an unnamed Collection - _if_ Geeqie is not already running.

The user interface for Collections is not optimal. Right-click on a thumbnail in the Collection window and select `View`. The main window image pane will then cycle through the contents of the Collection, and not the contents of the files pane.

The possibility for Collections to be displayed as a pseudo-directory is on the to-do list - but has been for a long time.

There appears to be bugs in the command line handling of --list-add, amongst other things.

**Comment by calestyo** [comment]
> Your command (or maybe with geeqie --list) will open the files as an unnamed Collection - if Geeqie is not already running. 

Yeah, but that then opens in a new window, which I guess cannot be docked, so I see then both the collection window with the thumbnails **and** the current directory section (with thumbnails) of the main window.

> The possibility for Collections to be displayed as a pseudo-directory is on the to-do list - but has been for a long time.

Is there already a ticket for this? If so, we could close this one as a duplicate of that.

**Comment by caclark** [comment]
#443 and this comment on freelists:
https://www.freelists.org/post/geeqie/Collections-as-folders

[The formatting of the email has gone badly wrong - but there is a link to an AppImage that is a demonstration of my idea for this]

Until now there has been no interest in this (apart from me).
